plugins/core/viame_train_detector.cxx

```diff
@@ -43,12 +43,18 @@
 #include <vital/algo/algorithm_factory.h>
 #include <vital/algo/train_detector.h>
 #include <vital/algo/detected_object_set_input.h>
+#include <vital/config/config_block_io.h>
+#include <arrows/ocv/image_container.h>
```

Comment:
The viame train detector app shouldn't directly require opencv or directly call the opencv image container - it should just be using the base vital algorithms

Reply:
Done.